### PR TITLE
Fixes #22513 - add exposed params to hiera

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
@ekohl Can I get a review please

File: config/foreman.hiera/tuning.yaml [hunk 1]
```diff
@@ -5,3 +5,11 @@ apache::mod::passenger::passenger_max_request_queue_size: 250
 apache::mod::passenger::passenger_stat_throttle_rate: 120
 apache::mod::passenger::passenger_max_requests: 10000
 apache::mod::passenger::passenger_min_instances: 1
+apache::mod::passenger::passengerMaxPreloaderIdleTime: 0
+apache::mod::passenger::passengerPassengerStartTimeout: 500
+apache::mod::prefork::startservers: 8
```
Comment (ekohl) [inline config/foreman.hiera/tuning.yaml]: These all match the defaults so I'm not sure we should add these.

File: config/foreman.hiera/tuning.yaml [hunk 2]
```diff
@@ -5,3 +5,11 @@ apache::mod::passenger::passenger_max_request_queue_size: 250
 apache::mod::passenger::passenger_stat_throttle_rate: 120
 apache::mod::passenger::passenger_max_requests: 10000
 apache::mod::passenger::passenger_min_instances: 1
+apache::mod::passenger::passengerMaxPreloaderIdleTime: 0
```
Comment (ekohl) [inline config/foreman.hiera/tuning.yaml]: CamelCase instead of snake_case

File: config/foreman.hiera/tuning.yaml [hunk 3]
```diff
@@ -5,3 +5,11 @@ apache::mod::passenger::passenger_max_request_queue_size: 250
...
+apache::mod::passenger::passengerPassengerStartTimeout: 500
```
Comment (ekohl) [inline config/foreman.hiera/tuning.yaml]: We override this in puppet-foreman and stopped using it in puppet-puppet so does this add anything?

**Review by ekohl — CHANGES_REQUESTED** [review]

File: config/foreman.hiera/custom.yaml [hunk 4]
```diff
@@ -15,3 +15,14 @@
 # performance or functionality of the application. Consider the impact of your
 # changes before applying them, and test them in a non-production environment
 # first.
+#
+# Here are some examples of how you tune the Apache and Passenger options if needed:
+#
+# apache::mod::passenger::passenger_max_preloader_idle_time: 0
+# apache::mod::passenger::passenger_passenger_start_timeout: 500
```
Comment (ekohl) [inline config/foreman.hiera/custom.yaml]: I think this parameter doesn't make sense anymore. puppet-foreman sets it explicitly and exposes it at a top level parameter.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: config/foreman.hiera/custom.yaml [hunk 5]
```diff
@@ -15,3 +15,12 @@
 # performance or functionality of the application. Consider the impact of your
 # changes before applying them, and test them in a non-production environment
 # first.
+#
+# Here are some examples of how you tune the Apache and Passenger options if needed:
```
Comment (ekohl) [inline config/foreman.hiera/custom.yaml]: There are no more Passenger options here

**Review by ekohl — CHANGES_REQUESTED** [review]

**Review by ekohl — APPROVED** [review]

**Comment by mmoll** [comment]
merged, thanks @chris1984!

**Pull request merged**